### PR TITLE
add CRUX to distrolist

### PR DESCRIPTION
**Pull request by TimB87**
For more information have a look over at https://crux.nu - I maintain this port and it's a public user repository that's listed in the official ports overlay [nnn in CRUX portdb](https://crux.nu/portdb/?a=search&q=nnn)
I'll keep it updated. Thanks for this great filemanager!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ It supports as many scripts as you need! Integrate utilities like sxiv (view ima
 - [Termux](https://github.com/termux/termux-packages/tree/master/packages/nnn) (`pkg in nnn`)
 - [Ubuntu](https://packages.ubuntu.com/search?keywords=nnn&searchon=names&exact=1) (`apt-get install nnn`)
 - [Void Linux](https://github.com/void-linux/void-packages/tree/master/srcpkgs/nnn) (`xbps-install -S nnn`)
+- [CRUX](https://github.com/TimB87/crux-ports/tree/master/nnn) (prt-get depinst nnn)
```
Comment (jarun) [inline README.md]: The list is alphabetical.

Also put the command within backquotes.

Comment (TimB87) [reply README.md]: Woops, pushed the requested changes :+1:

**Comment by jarun** [comment]
Thank you for your contribution.

**Comment by jarun** [comment]
Is it a official CRUX package? I don't see it listed here for CRUX: https://repology.org/project/nnn/versions

If not, can you please submit it to the distro?

**Comment by TimB87** [comment]
Hey, I can try to do this, but I can't promise anything. Right now, it's just my repo, but it's very easy to activate.

**Comment by jarun** [comment]
Just give it a shot!

**Comment by TimB87** [comment]
Listen, this is my ultimate plan after all, but I have some stuff to prepare first. But will do.

**Comment by jarun** [comment]
Sure thing! Nothing to hurry.

**Comment by TimB87** [comment]
Hey @jarun 
It's done :) 
https://crux.nu/gitweb/?p=ports/contrib.git;a=commit;h=ec34728b9e0402bcdccadb9ba5059a4eaf739ce1
Great work with the latest update, thanks a lot for the hard work!

**Comment by jarun** [comment]
Thank you for the update!

**Pull request merged**